{
    "pattern": "DROP TABLE IF EXISTS u; DROP TABLE IF EXISTS ff1 <on_cluster_parameter>; DROP TABLE IF EXISTS d <on_cluster_parameter>; CREATE TABLE ff1 <on_cluster_parameter> (uid Int16) ENGINE = <engine_parameter> ORDER BY <order_parameter> AS SELECT 1; CREATE TABLE d <on_cluster_parameter> AS ff1 ENGINE = <engine_parameter>; CREATE TABLE u(uid Int16) ENGINE = <engine_parameter> AS SELECT 1; SELECT <select_parameter> FROM d WHERE uid <global_operator_replace> (SELECT <select_parameter> FROM u); SELECT <select_parameter> FROM d WHERE uid <global_operator_replace> u SETTINGS <settings_parameter>; SELECT <select_parameter> FROM d WHERE uid <global_operator_replace> u SETTINGS <settings_parameter>;",
    "applied_rules": [
        "<on_cluster_parameter>",
        "<engine_parameter>",
        "<order_parameter>",
        "<select_parameter>",
        "<global_operator_replace>",
        "<settings_parameter>"
    ],
    "Affected Elements": "GLOBAL IN clause",
    "Root Cause Analysis": "The error is due to the inability of the SQL engine to identify the 'default.u' table, which is likely caused by the distributed query context not correctly resolving the identifier."
}